contrib/devtools/github-merge.py

```diff
@@ -175,6 +175,7 @@ def main():
     if info is None:
         exit(1)
     title = info['title'].strip()
+    body = '  ' + info['body'].replace('\n', '\n  ')
```

Comment:
I think here should be `body = info['body'].strip()` like the above. And the body indentation should be below.